up-and-running/auth/README.md

```diff
@@ -0,0 +1,23 @@
+# Authentication
+
+Rather than handling user accounts itself, npm Enterprise allows
+you to connect to your company's existing authentication system.
+
+Currently we support:
+
+- [Open Authentication]
+- [GitHub Enterprise]
+- Bitbucket Cloud
+- LDAP
+- SAML
+- OAuth 2
+
+## Writing Custom Authentication Plugins
+
+You can also opt to write your own [custom Auth-Plugin] for npm Enterprise.
+
+Install your module on the host machine in the `Miscellaneous data files` storage directory (this defaults to `/usr/local/lib/npme/data`). The module will be available at `/etc/npme/data` inside your container.
```

Comment:
We can reference https://github.com/nexdrew/npme-auth-gitlab as an example custom plugin.